src/checks.rs

```diff
@@ -1444,6 +1449,9 @@ impl CheckKind {
             CheckKind::UnnecessaryFutureImport(name) => {
                 format!("Unnessary __future__ import `{name}` for target Python version")
             }
+            CheckKind::UnnecessaryLRUCacheParams => {
+                "Unnessary parameters to functool.lru_cache".to_string()
```

Comment:
Cool stuff. Just a small nitpick: 

```suggestion
                "Unnessary parameters to functools.lru_cache".to_string()
```